src/ucp/api/ucp_def.h

```diff
@@ -289,6 +299,17 @@ typedef void (*ucp_send_callback_t)(void *request, ucs_status_t status);
 typedef void (*ucp_err_handler_cb_t)(void *arg, ucp_ep_h ep, ucs_status_t status);
 
 
+/**
+* @ingroup UCP_WORKER
+* @brief A callback for handling the creation of an endpoint (connection)
+*        to the remote peer after an incoming connection request on the listener.
+*
+*  @param [in]  ep_h      A handle to the created endpoint.
```

Comment:
s/ep_h/ep/
Handle to a newly created endpoint which is connected to the remote peer which has initiated the connection